integration-tests/implementationGuides.test.ts

```diff
@@ -94,7 +95,14 @@ describe('Implementation Guides - US Core', () => {
         );
 
         const expectedCapStatement: CapabilityStatement = (
-            await axios.get(`https://www.hl7.org/fhir/us/core/${usCoreVersion}/CapabilityStatement-us-core-server.json`)
+            await axios.get(
+                `https://www.hl7.org/fhir/us/core/${usCoreVersion}/CapabilityStatement-us-core-server.json`,
+                {
+                    httpsAgent: new https.Agent({
+                        rejectUnauthorized: false,
```

Comment:
It's probably better to download the capability statement from https://www.hl7.org/fhir/us/core/STU3.1.1/CapabilityStatement-us-core-server.json and commit the file as part of the integ tests code. That gets rid of any other transient issue (i.e. www.hl7.org is down) and the json files are not that big.

Plus, disabling certificate validation is an odd choice that will be flagged by any security scan. (CodeQL already flagged it). We can dismiss it as "used in tests" but that's less than ideal

Reply:
That makes sense. I just updated the code to use the downloaded JSON.